packages/cli-e2e/__tests__/atomic.specs.ts

```diff
@@ -169,15 +169,6 @@ describe('ui:create:atomic', () => {
       buildAppOptions: {id: 'without-page-id', skipInstall: true},
       skipBrowser: true,
     },
-    {
-      describeName: 'when using an existing pageId (--pageId flag specified)',
-      buildAppOptions: {
-        id: 'with-page-id',
-        pageId: 'fffaafcc-6863-46cb-aca3-97522fcc0f5d',
-        skipInstall: true,
-      },
-      skipBrowser: true,
-    },
```

Comment:
So we stop testing that? :suspect:

Reply:
When I looked at it last year it seemed like a duplicate. Now I'm not so sure...

Reply:
After DM, yep you're right it's a dupe.
LGTM!